components/ui/navigation-menu.tsx

```diff
@@ -41,7 +41,7 @@ NavigationMenuList.displayName = NavigationMenuPrimitive.List.displayName
 const NavigationMenuItem = NavigationMenuPrimitive.Item
```

Comment:
Althought this is a component from shadcn, I made a few changes here since we are 99% not going to use this comp anywhere else but the header. Even in the situation we do use it somewhere else it is likely we would want to keep these color styles and themeings.